### PR TITLE
Added method getExtraContent

### PR DESCRIPTION
**Pull request by MaximPlusov**

### CODE REVIEW

File: org.verapdf.releng/pom.xml [hunk 1]
```diff
@@ -104,6 +104,13 @@
 					<groupId>org.eclipse.xtend</groupId>
 					<artifactId>xtend-maven-plugin</artifactId>
 					<version>${xtext.version}</version>
+					<dependencies>
+						<dependency>
+							<groupId>org.eclipse.platform</groupId>
+							<artifactId>org.eclipse.equinox.common</artifactId>
+							<version>3.10.200</version>
```
Comment (carlwilson) [inline org.verapdf.releng/pom.xml]: Can we make this a single maven property please so we only change the version in one place, thanks.

File: org.verapdf.tests/pom.xml [hunk 2]
```diff
@@ -30,6 +30,13 @@
 			<plugin>
 				<groupId>org.eclipse.xtend</groupId>
 				<artifactId>xtend-maven-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.platform</groupId>
+						<artifactId>org.eclipse.equinox.common</artifactId>
+						<version>3.10.200</version>
```
Comment (carlwilson) [inline org.verapdf.tests/pom.xml]: See property comment.

File: org.verapdf.ui/pom.xml [hunk 3]
```diff
@@ -30,6 +30,13 @@
 			<plugin>
 				<groupId>org.eclipse.xtend</groupId>
 				<artifactId>xtend-maven-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.platform</groupId>
+						<artifactId>org.eclipse.equinox.common</artifactId>
+						<version>3.10.200</version>
```
Comment (carlwilson) [inline org.verapdf.ui/pom.xml]: and here please :)

File: org.verapdf/pom.xml [hunk 4]
```diff
@@ -87,6 +87,13 @@
 			<plugin>
 				<groupId>org.eclipse.xtend</groupId>
 				<artifactId>xtend-maven-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.platform</groupId>
+						<artifactId>org.eclipse.equinox.common</artifactId>
+						<version>3.10.200</version>
```
Comment (carlwilson) [inline org.verapdf/pom.xml]: once more.

File: .travis.yml [hunk 5]
```diff
@@ -3,8 +3,8 @@ sudo: required
 dist: trusty
 
 jdk:
-  - oraclejdk7
```
Comment (carlwilson) [inline .travis.yml:6]: and maybe go openJDK here if possible.

**Review by carlwilson — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 5 above)
Comment (MaximPlusov) [reply .travis.yml:6]: openjdk8?

**Review by carlwilson — COMMENTED** [review]
good stuff

**Review by carlwilson — APPROVED** [review]

**Pull request merged**